pages/product-listing/product-listing.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jewelory</title>
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="./product-listing.css">
+    <link rel="stylesheet" href="https://aems-ui.netlify.app/aems.css" />
+    <script src="https://kit.fontawesome.com/e24bb32a8e.js" crossorigin="anonymous"></script>
+
+
+
```

Comment:
Remove extra spacing.

Reply:
you can use beautify extension to make code look beautiful.